nodes/test/guardrails/test_all.py

```
@@ -0,0 +1,938 @@
+            'enable_pii_detection',
+            'enable_hallucination_check',
+            'max_input_length',
+            'max_tokens_estimate',
+            'blocked_topics',
+            'allowed_topics',
+            'expected_format',
+        }
+
+        # Collect all keys from all profiles
+        profile_keys = set()
+        for profile_name, profile_config in profiles.items():
```

Ruff (0.15.7)

[warning] Loop control variable `profile_name` not used within loop body

Rename unused `profile_name` to `_profile_name`

(B007)